main.css

```diff
@@ -0,0 +1,368 @@
+body{
+    padding: 0 calc(50% - 460px);
+    font-family: sans-serif;
+    font-stretch: ultra-condensed;
+    color: rgb(114, 110, 110);
+}
+
+header{
+    display: flex;
+    justify-content: space-between;
+    padding: 10px;
+}
+
+.logo{
+    font-size: 80%;
+    padding: 2%;
+}
+
+.logo h1{
+    margin-bottom: 0px;
+}
+
+.logo p{
+    margin-top: 0px;
+}
+
+nav{
+    width: 60%;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 2%;
+}
+
+nav a{
+    padding: 5% 3% 5% 3%;
+    text-decoration: none;
+    color: rgb(112, 109, 109);
+    text-align: center;
+}
+
+nav :first-child{
+    background-color: rgb(224, 105, 115);
+    color: white;
+    border-radius: 10%;
+}
+
+/* ------------------------------MAIN IMAGE------------------------------------------------------- */
+
+.img_table {
+    background: url(img/main.jpg) center/cover no-repeat;
+    width: 100%;
+    height: 300px;
+    border-spacing: 0;
+}
+
+.img_table td{
+    vertical-align: bottom;
+    width: 50%;
+    height: 50%;
+    padding-bottom: 6px;
+}
+
+.img_table tr:first-child td:first-child{
+    background-color:rgba(0, 0, 0, .6);
+}
+
+.img_table tr:first-child td:last-child{
+    background-color:rgba(0, 0, 0, .4);
+}
+
+.img_table tr:last-child td:first-child{
+    background-color:rgba(0, 0, 0, .2);
+}
+
+.img_table td p{
+
+    color: white;
+    margin: 0px;
+    padding: 5px;
+    display: inline;
+    opacity: 1;
+    background-color:black;
+}
+
+.img_table tr:last-child td:last-child p{
+    background-color: rgb(182, 83, 83);
+}
+
+/* ------------------------------MORE INFO------------------------------------------------------- */
+
+.section_more_info{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    padding: 30px 0px;
+}
+
+.section_more_info .btn{
+    font-size: 16px;
+    padding: 16px;
+}
+
+.section_more_info div h1{
+    margin-bottom: 0px;
+}
+
+.section_more_info div p{
+    margin-top: 0px;
+}
+
+/* ------------------------------ITEMS------------------------------------------------------- */
+
+.section_items{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+
+.div_item{
+    display: flex; 
+    justify-content: space-between;
+    box-sizing: border-box;
+    width: 50%;
+    padding: 2% 3%;
+}
+
+.div_item img{
+    align-self: center;
+    width: 30px;
+    height: 30px;
+    padding: 30px;
+    border-radius: 100%;
+    border: 3px solid rgb(201, 201, 201);
+}
+
+.div_item div{
+    padding: 0 5%;
+}
+
+.div_item div h4{
+    margin-top: 10%;
+    margin-bottom: 0px;
+}
+
+.div_item div p{
+    margin-top: 1%;
+    font-size: 13px;
+    font-family: 'Times New Roman', Times, serif;
```

Comment:
try to use the font from PSD (or if you will not find it on the internet, try to include some not standard font)